### ISSUE TITLE
[Learn] Learning Paths need enhanced with Meshery/MeshMap walk-throughs

### ISSUE THREAD

**Issue by leecalcote** [layer5io/layer5#4899]
**Current State:**

The [Learning Path Labs](https://layer5.io/learn/service-mesh-labs) are an excellent resource for DevOps engineers and others training on cloud native technologies. Each of the learning path labs has multiple labs for specific techonolgy, and each labs has any number of scenarios. Within each scenario, students/users will find hands-on labs. The hands-on labs portions are out-of-date.

**Desired State:**
The hands-on labs need to have their embedded Katacoda scenarios replaced with videos of Meshery showing how to do what is in the lab. The Meshery Playground can be used to create such recordings. The recordings can be uploaded to the [YouTube channel](https://www.youtube.com/@Layer5io).

**Checklist for scenarios**
All the scenarios need to be replaced using above mentioned desired state.

- [x] #4900
- [x] #4901
- [x] #4902
- [x] #4903
- [x] #4904
- [x] #4905
- [x] #4906
---
**Contributor Resources**
The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- Check the Learn Layer5 site [Design Spec](https://docs.google.com/document/d/1tQ9tLbetBVKP7z76FR19vsxbAfoFHZzZYxCt7lJpAhU/edit) to know more about the site structure and development plans.
- Checkout the Learn Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/?node-id=4853%3A0).

**Comment by abhijeetgauravm** [comment]
there u go @sandramsc

**Comment by goldfishdolphin** [comment]
Hi @sandramsc 
Have you started working on it? We can collaborate on this issue.

**Comment by sandramsc** [comment]
> Hi @sandramsc Have you started working on it? We can collaborate on this issue.

Hi, yes I have, sure. I suggest we start a thread in Slack so as to reduce the "noise" (notification wise)  by planning how to reslove the issue on here?

**Comment by Sgiriarun** [comment]
hi @sandramsc @goldfishdolphin can i join with u guys, for resolving issues.

**Comment by sandramsc** [comment]
> hi @sandramsc @goldfishdolphin can i join with u guys, for resolving issues.

Hi, what is your Slack handle within the community so I can tag you in the conversation, then we pick it up from there and see if you're still up for it?

**Comment by Sgiriarun** [comment]
> > hi @sandramsc @goldfishdolphin can i join with u guys, for resolving issues.
> Hi, what is your Slack handle within the community so I can tag you in the conversation, then we pick it up from there and see if you're still up for it?

sgiriarun@gmail.com 
this is my mail-id, u guys can add to your community.

**Comment by sandramsc** [comment]
> > > hi @sandramsc @goldfishdolphin can i join with u guys, for resolving issues.
> > > Hi, what is your Slack handle within the community so I can tag you in the conversation, then we pick it up from there and see if you're still up for it?
> 
> [sgiriarun@gmail.com](mailto:sgiriarun@gmail.com) this is my mail-id, u guys can add to your community.

Hi, the request inquired for your Slack username, not email. If you're not already part of the Layer5 community on Slack consider joining there and continuing this conversation on there to reduce the "noise" on this issue. Also please see previouse commments regarding.

Thanks.

**Comment by Sgiriarun** [comment]
yeah i have joined. we work there and please mentor me @sandramsc @goldfishdolphin

**Comment by saurabh100ni** [comment]
Any updates @Sgiriarun @sandramsc @goldfishdolphin?

**Comment by sandramsc** [comment]
Unassigning self from issue:
- was able to install and run meshery locally using microk8s
- however, the installation was temporary as after a short while cluster would not restart 
- this could be due to my machines' phyical capacity
- please see discussions [here](https://layer5io.slack.com/archives/CFGG6U10E/p1700482860411539?thread_ts=1700473197.362409&cid=CFGG6U10E) first to grasp progress before for picking up the issue

**Comment by saurabh100ni** [comment]
@goldfishdolphin are you working on this issue? if not, we can give opportunity to new people to work on this.

**Comment by goldfishdolphin** [comment]
@saurabh100ni  can you please assign it to someone?

**Comment by saurabh100ni** [comment]
Sure thing 🙌

**Comment by Shubhf** [comment]
@leecalcote Can I work on this issue?

**Comment by Sumedh-6504** [comment]
Hi, I would Like to work on this issue! Please assign me
/take

**Comment by Maanvi212006** [comment]
Closing as all sub-issues completed!
